CHANGELOG.rst

```diff
@@ -2,6 +2,183 @@
 Release Notes
 =============
 
+.. _Release Notes_7.28.0:
+
+7.28.0 / 6.28.0
+======
+
+.. _Release Notes_7.28.0_Prelude:
+
+Prelude
+-------
+
+Release on: 2021-05-20
```

Comment:
We should leave this to `unreleased` until the actual release date (would make sense to update the changelog generation script accordingly)